Common/CUDA/tvdenoising.cu

```diff
@@ -436,6 +444,10 @@ void cpy_from_host(float* device_array,float* host_array,
                         cudaStreamSynchronize(stream[dev*nStream_device+4]);
                         if (is_last_chunk) {cudaMemsetAsync(d_pz[dev], 0, mem_img_each_GPU,stream[dev*nStream_device+4]);}
                         cpy_from_host(d_pz[dev],h_pz,bytes_device[dev], offset_device[dev],offset_host[dev], pixels_per_slice, buffer_length, stream[dev*nStream_device+4],  is_first_chunk,  is_last_chunk, image_size);
+                        // Z derivative must be negated in sign to keep Neumman conditions

```

Comment:
The code has a bit of a error. here is the new codes. 
```
if (is_first_chunk){
    multiplyArrayScalar<<<60,MAXTREADS,0,stream[dev*nStream_device+4]>>>(d_pz[dev],             -1,  pixels_per_slice*buffer_length);     
}
if (is_last_chunk){
    multiplyArrayScalar<<<60,MAXTREADS,0,stream[dev*nStream_device+4]>>>(d_pz[dev]+bytes_device[dev],-1,  pixels_per_slice*buffer_length);   
}
```
I have test the code, and here is the results
![image](https://github.com/user-attachments/assets/d9765209-657a-4364-a322-cff35d4a6e7d)
the left image: using no negative of pz, middle image: Negative operations on the cpu， right: Negative operations on the gpu(using this new code), there is a little difference between cpu and gpu, why？

Reply:
hum I don't really know, you are right that it should be the same...

Reply:
@zezisme I don't really understand the difference, but I also don't see anything wrong in the code anyway... Given the GPU one strangely is doing better (less SD, mean value closer to original) I will fix the little bug and merge this. Thank a lot for the help so far :)